doc/source/theory/fluid_dynamics/linear_solvers.rst

```diff
@@ -1,4 +1,154 @@
 Linear solvers
 ##############
 
-**Under construction**
\ No newline at end of file
+After discretization in space and time of the Navier-Stokes equations, we obtain a problem that needs to be solved using a Netwon non-linear solver that solves a linear system in every iteration. In the case of a stabilized approach, we need to solve a linear system that looks as follows:
+
+.. math::
+
+    \underbrace{\left[ \begin{matrix} 	A^* & B^{*T}  \\[0.3em]	B^* & S^* \end{matrix} \right]}_{\mathcal{A}} \underbrace{\left[ \begin{matrix} \delta \textbf{u}   \\[0.3em]		\delta p  \end{matrix} \right]}_{x}  &=  \underbrace{-\left[ \begin{matrix} \mathbf{\mathcal{R}}_{v}^*   \\[0.3em]		\mathbf{\mathcal{R}}_{q}^*  \end{matrix} \right]}_{b}
+
+where the matrix to the left-hand side is non-symmetric. In the literature this is frequently called a *generalized saddle point problem*. 
+
+In Lethe we support the following linear solvers:
+
+* Trilinos direct solvers.
+* GMRES preconditioned with ILU or AMG.
+* BiCGStab preconditioned with ILU.
+
+Only coupled methods are used at the moment: a direct method and two iterative algorithms (Krylov subspace methods). We are mainly interested in solving problems that are both large and sparse, therefore, the two iterative methods will be explained in this section, along with their preconditioners. For more information on the different direct solvers supported by Trilinos see the `deal.II <https://www.dealii.org/>`_ documentation: `TrilinosWrappers::SolverDirect <https://www.dealii.org/current/doxygen/deal.II/classTrilinosWrappers_1_1SolverDirect.html>`_. **This is not by any means a detailed explanation of all the linear solvers; however, it should give you a general idea and it should point you to useful references in case you are more interested in this topic.**
```

Comment:
I would add that the direct solver should only be used for tests and in the development of features since it is extremely slow.

Reply:
There is only one direct solver, I would re-phrase the sentence:'' For more information on the different direct solvers supported by Trilinos''